collector/exchange.go

```diff
@@ -0,0 +1,679 @@
+// +build windows
+
+package collector
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/StackExchange/wmi"
```

Comment:
Did you check if these classes are available as Performance Counters that we could read with perflib instead? That would be much preferred from a performance point of view.

Reply:
Hello and thanks for the review. 

I havent really looked into perlib much at all, but I'm willing to do so. Any way to quickly establish if there are perflib equivalents?

Reply:
Hey,
The easiest way is probably "can you find it in perfmon?". The name of the counter object is the "heading" in perfmon, so "Processor Information" for example here:
![image](https://user-images.githubusercontent.com/214867/80921936-eb82d280-8d79-11ea-9955-496865fbc880.png)

I whipped up a small tool to look at a counter object here: https://github.com/carlpett/perfcounter-inspector
You can clone that and do `go run . <name of counter>` to get a dump of what fields there are, and a sample of the values. Might be useful? 
(I will rewrite the collector generator with this approach)

Reply:
Ok, so Robin has changed from WMI to perflib, and we have walked through every metric and to the best of our knowledge they now have the correct type set. Can you have a look?